training/src/test/java/wolox/training/utils/Utils.java

```diff
@@ -0,0 +1,30 @@
+package wolox.training.utils;
+
+public class Utils {
+
+    public static final String BODY_WITH_USER =
+        "{\"username\": \"guille1234\", \"name\": \"guille\""
+            + ", \"birthdate\": \"1993-12-06\"}";
+    public static final String BODY_WITH_BOOK_UPDATED =
+        "{\"id\": \"1\", \"year\": \"1990\""
+            + ", \"title\": \"el señor de los anillos\""
+            + ", \"subtitle\": \"las dos torres\""
+            + ", \"pages\": \"30\""
+            + ", \"publisher\": \"epublisher 1234\""
+            + ", \"genre\": \"ficcion\""
+            + ", \"author\": \"un autor\""
+            + ", \"isbn\": \"isgn 12\""
+            + ", \"image\": \"one image\""
+            + "}";
+    public static final String BODY_WITH_BOOK_CREATED =
```

Comment:
What are these used for? 🤔

Reply:
I need this Utils to specify the content of body in the request:

![image](https://user-images.githubusercontent.com/42941989/84190197-ebf34500-aa6c-11ea-9991-07a3bbf534ba.png)